imports/plugins/included/product-admin/client/hocs/withProduct.js

```diff
@@ -138,21 +134,6 @@ const wrapComponent = (Comp) => {
           const { newVariantId } = await handleCreateVariant(product);
           history.push(`/operator/products/${product._id}/${newVariantId}`);
         }}
-        onMetaChange={setNewMetaField}
-        onMetaRemove={handleMetaRemove}
-        onMetaSave={(productId, metafield, index) => {
-          // update existing metafield
-          if (index >= 0) {
-            Meteor.call("products/updateMetaFields", productId, metafield, index);
-          } else if (metafield.key && metafield.value) {
-            Meteor.call("products/updateMetaFields", productId, metafield);
-          }
-
-          setNewMetaField({
-            key: "",
-            value: ""
-          });
-        }}
```

Comment:
Did you intend to remove all this? Seems like `onMetaSave` is still called by the metadata component.

Reply:
This was moved to the `withProductForm` HOC

Reply:
👍 I missed that but I see it now.